### DISCUSSION TITLE
Any ways to enable Tex Gyre math fonts?

### DISCUSSION THREAD

**Discussion by andyluooo** [KaTeX/KaTeX#3822] [Q&A]
Are there any ways to change fonts of Katex? Or any future plans to enable other math fonts? (e.g. Tex Gyre Pagella Math)

**Comment by sxlxc** [comment]
This blog post may help <https://yingtongli.me/blog/2022/09/24/katex-custom-fonts.html>. I tried it before. It doesn't look perfect. 
<img width="119" alt="image" src="https://github.com/KaTeX/KaTeX/assets/68147938/ae4603d1-28bc-4fde-9a87-ddd13477d2ad">(this is Tex Gyre Pagella Math font)
